migrations/20240117151749_init_db.up.sql

```diff
@@ -19,13 +25,6 @@ CREATE TABLE IF NOT EXISTS legal_persons
     adoptionPolicy longtext    not null
 );
 
-CREATE TABLE IF NOT EXISTS persons
-(
-    id        uuid primary key default UUID(),
-    userId    uuid REFERENCES users (id),
-    birthdate date not null
-);
-
 CREATE TABLE IF NOT EXISTS addresses
```

Comment:
O ideal para alteração na base de dados usando migrations é que se criar um nova para qualquer tipo de alteração. Dessa forma não irá impactar os bancos de dados que foram criados usando esse arquivo na sua forma inicial.

Reply:
Aqui nos combinamos que enquanto nao for a producao nao vale a pena ficar criando, mas apos isso sim iremos adotar